falco.yaml

```diff
@@ -170,6 +170,12 @@ syscall_event_drops:
 syscall_event_timeouts:
   max_consecutives: 1000
 
+# Enabling this option allows Falco to drop failed syscalls exit events.
+# This can enable some small optimization both in CPU usage and ring buffer usage,
+# possibly leading to lower number of event losses.
+# Be careful: enabling it also means losing a bit of visibility on the system.
+syscall_drop_failed: false
```

Comment:
Suggestion: `drop_failed_syscall_exit`

Reply:
Similar to below event, perhaps add more details around where we drop them, e.g.
`Enabling this option allows Falco to drop failed syscall exit events in the kernel driver before the event is pushed onto the ring buffer.`

Reply:
> drop_failed_syscall_exit

I'd leave the `syscall` name as prefix, as it is the `macro` argument (and we already have eg: `syscall_event_timeouts`).

Reply:
`syscall_drop_failed_exit`?

Reply:
Done!